app/controllers/ProcessReviewController.scala

```diff
@@ -62,13 +63,13 @@ class ProcessReviewController @Inject() (reviewService: ReviewService, cc: Contr
   def approval2iReviewComplete(id: String): Action[JsValue] = Action.async(parse.json) { request =>
     def save(statusChangeInfo: ApprovalProcessStatusChange): Future[Result] = {
       reviewService.twoEyeReviewComplete(id, statusChangeInfo).map {
-        case Right(_) => NoContent
+        case Right(ap) => getAuditInfo(statusChangeInfo.userId, ap)
```

Comment:
It doesn't look like we are using user.name at present, but I think it may be used when we implement comments.

Reply:
Not using yet, no real requirements for auditing tbh, so hopefully implementing comments will sort this out